input/community/speakers/maarten-balliauw.md

```diff
@@ -0,0 +1,24 @@
+---
+Title: Maarten Balliauw
+Location: Antwerp, Belgium
+Email: maarten@maartenballiauw.be
+Language:
+  - English
+Topics:
+  - .NET
+  - ASP.NET MVC
+  - Architecture
+  - Azure
+  - C#
+  - NuGet
+  - Open Source
+  - Visual Studio
+  - ReSharper
+  - JetBrains Rider
```

Comment:
```suggestion
```

These are custom entries that are added to the list. Suggesting the removal of them so that the build passes (though I love the idea of adding these to the list in a separate PR!)

Reply:
@SeanKilleen That PR exists :) #595

Reply:
Ah :) Sorry I'd missed it!

Reply:
No worries :-) Thanks!

Reply:
Seems that one got merged, woohoo!